src/models/helpers/admin.rs

```diff
@@ -0,0 +1,91 @@
+use std::{collections::HashSet, num::ParseIntError, str::FromStr};
+
+use lazy_static::lazy_static;
+
+use crate::util::errors::{forbidden, AppResult};
+
+lazy_static! {
+    static ref AUTHORIZED_ADMIN_USER_IDS: HashSet<i32> =
+        parse_authorized_admin_users(dotenv::var("GH_ADMIN_USER_IDS"));
+}
+
+// The defaults correspond to the current crates.io team.
+//
+// FIXME: this needs to be removed once we can detect the admins from the Rust teams API.
+const DEFAULT_ADMIN_USER_IDS: [i32; 5] = [193874, 22186, 141300, 229984, 20070360];
```

Comment:
might be best to add the GitHub usernames in comments next to those magic numbers

Reply:
Yep, totally forgot to do that before pushing. Thanks!